backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -123,4 +123,27 @@ private void verifyManagementRole(String authString) {
     verifyManagementRole(authString);
     return staffService.getActiveStaff();
   }
+
+  /**
+   * Endpoint for the manager to remove a staff member by a given id.
+   * 
+   * @param authString manager's authentication token.
+   * @param id staff id to remove.
+   * @return An updated list of active staff members.
+   */
+  @GetMapping("/staff/remove")
```

Comment:
If the purpose of this endpoint is to update an object (change status to 'inactive') then this should be a PUT, rather than a GET.

Reply:
fixed